<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="translateField">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>201</width>
      <height>38</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>translate value...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rotateField">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>135</y>
      <width>201</width>
      <height>38</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>rotate value...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scaleField">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>195</y>
      <width>201</width>
      <height>38</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>scale value...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="translateAx">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>51</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(154, 153, 150);
	padding-left: 10px;
	font: 300 11pt &quot;URW Bookman&quot;;
}

 QComboBox:on { /* сдвиг текста когда выпадающий список раскрывается */
    padding-left: 20px;
 }</string>
    </property>
    <property name="currentText">
     <string>X</string>
    </property>
    <item>
     <property name="text">
      <string>X</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="rotateAx">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>135</y>
      <width>51</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(154, 153, 150);
	padding-left: 10px;
	font: 300 11pt &quot;URW Bookman&quot;;
}

 QComboBox:on { /* сдвиг текста когда выпадающий список раскрывается */
     padding-left: 20px;
 }</string>
    </property>
    <property name="currentText">
     <string>X</string>
    </property>
    <item>
     <property name="text">
      <string>X</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="scaleButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>195</y>
      <width>81</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>135</y>
      <width>81</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="translateButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>81</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>661</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="fileInfoTop">
       <item>
        <widget class="QPushButton" name="fileDialogButton">
         <property name="font">
          <font>
           <family>URW Bookman</family>
          </font>
         </property>
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fileNameLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="vLable">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>URW Bookman</family>
          </font>
         </property>
         <property name="text">
          <string>Vertices</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="vLabelCount">
         <property name="minimumSize">
          <size>
           <width>67</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>URW Bookman</family>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="eLabel">
         <property name="font">
          <font>
           <family>URW Bookman</family>
          </font>
         </property>
         <property name="text">
          <string>Edges</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="eLabelCount">
         <property name="font">
          <font>
           <family>URW Bookman</family>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>551</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="openGLlayout"/>
   </widget>
   <widget class="QPushButton" name="vertex_empty">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>540</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertex_circle">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>540</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertex_square">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>703</x>
      <y>510</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Type vertex</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edg_line">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>470</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edg_dotted">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>470</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="text">
     <string>Dotted</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>661</x>
      <y>440</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Type_edg</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Color_B_V_E">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(154, 153, 150);
	padding-left:5px;
	font: 300 11pt &quot;URW Bookman&quot;;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Backround</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vertex</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Edg</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="slider_green">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>399</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_blue">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_red">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>377</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_size_v_e">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>337</y>
      <width>180</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>312</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: &quot;URW Bookman&quot;;
}</string>
    </property>
    <property name="text">
     <string>Size_vertex_or_edg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>354</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="scaleAx">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>195</y>
      <width>51</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(154, 153, 150);
	padding-left: 10px;
	font: 300 11pt &quot;URW Bookman&quot;;
}

 QComboBox:on { /* сдвиг текста когда выпадающий список раскрывается */
     padding-left: 10px;
 }</string>
    </property>
    <property name="currentText">
     <string>X</string>
    </property>
    <item>
     <property name="text">
      <string>X</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="button_screenshot">
    <property name="geometry">
     <rect>
      <x>839</x>
      <y>260</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
     </font>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <widget class="QComboBox" name="format_screenshot">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>260</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(154, 153, 150);
	padding-left: 10px;
	font: 300 11pt &quot;URW Bookman&quot;;
}</string>
    </property>
    <property name="currentText">
     <string>jpeg</string>
    </property>
    <item>
     <property name="text">
      <string>jpeg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bmp</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="type_projection">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>140</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(154, 153, 150);
	padding-left: center;
	font: 300 11pt &quot;URW Bookman&quot;;
}

 QComboBox:on { /* сдвиг текста когда выпадающий список раскрывается */
     padding-left: center;
 }


</string>
    </property>
    <item>
     <property name="text">
      <string>Parallel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Central</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="button_gif">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
